src/backend/libc/shm/types.rs

```diff
@@ -2,9 +2,9 @@ use crate::backend::c;
 use bitflags::bitflags;
 
 bitflags! {
-    /// `O_*` constants for use with [`shm_open`].
+    /// `O_*` constants for use with [`open`].
```

Comment:
I think it'd help avoid confusion to change the text here from \`open\` to \`shm::open\`. I think that matches how users will want to use it, as `use rustix::shm; ... shm::open` rather than `use rustix::shm::open; ... open`.

And similar for \`init\` and other functions in this PR.

Reply:
Seems like a good improvement! This confusion is part of the reason I prefer `shm_open` (I also think autocomplete support is better with just the function name to deal with), but if we display all uses in the docs with the module prefix then that probably gets us most of the way there.

Reply:
Yeah ok this looks a lot better now, nice!